umh-core/examples/example-config-comm.yaml

```
@@ -19,12 +19,23 @@ agent:
-  desiredState: stopped
+dataFlow:
+  - name: hello-world-dfc
+    desiredState: active
+    benthosServiceConfig:
+      input:
+        generate:
+          mapping: root = "hello world from Benthos!"
+          interval: 1s # Output every 2 seconds
+          count: 0 # Set to 0 for unlimited messages
+      output:
+          stdout: {}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)